pkg/tagger/utils/taglist.go

```diff
@@ -74,8 +74,8 @@ func (l *TagList) AddAuto(name, value string) {
 	l.AddLow(name, value)
 }
 
-// Compute returns two string arrays in the format "tag:value"
-// first array is low cardinality tags, second is high card ones
+// Compute returns three string arrays in the format ["tag0:value0", "tag1:value1"]:
```

Comment:
Should we convert return value into a struct?

Reply:
perhaps i'll make a follow up PR to update this everywhere, OK with you?

Reply:
yes, sure, np